src/platform/site-wide/announcements/components/VAPlusVetsModal.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation/Modal';
+import brandConsolidation from '../../../brand-consolidation';
+
+export default class VAPlusVetsModal extends React.Component {
+  static isEnabled() {
+    if (!brandConsolidation.isEnabled()) return false;
+
+    const referrer = document.referrer;
+    let wasRedirectedFromVets = !!referrer && referrer.includes('vets.gov');
+
+    // Allow an override on the URL to force the Onboarding Modal to appear for testing purposes.s
+    if (__BUILDTYPE__ !== 'preview') {
+      wasRedirectedFromVets = window.location.search.includes(
+        'onboarding-modal',
+      );
+    }
```

Comment:
This isn’t really the way the enabled property was intended to work for Announcements, but because this modal should take priority over all others and appear every time, it was a weird situation.